packages/mui-base/src/PopperUnstyled/PopperUnstyled.js

```diff
@@ -84,7 +84,7 @@ const PopperTooltip = React.forwardRef(function PopperTooltip(props, ref) {
    * modifiers.flip is essentially a flip for controlled/uncontrolled behavior
    */
   const [placement, setPlacement] = React.useState(rtlPlacement);
-  const [tooltipAnchorEl, setTooltipAnchorEl] = React.useState(anchorEl);
+  const [tooltipAnchorEl, setTooltipAnchorEl] = React.useState(resolveAnchorEl(anchorEl));
```

Comment:
Since this is always a resolved element, calling it `resolvedAnchorEl` (or even `resolvedAnchorElement`, to reduce abbreviations), would make the intents clearer